### ISSUE TITLE
Do not remove a Source-User association but make it inactive

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-base/ManagementPortal#219]
The REST-API would like to have a historical record on what devices were associated to what users. One way to do this is to add an `active` column to the table associating sources to users. Whenever the association is broken, the column value would be set to `false`. The JWT would not contain inactive sources, and the REST API could expose the active property, e.g.
`GET /users/sub-1`
```json
{
  ...,
  "login": "sub-1",
  ...,
  "sources": {
    ...
  },
  "inactiveSources": {
  }
}
```

This is similar to the request in #218.

**Comment by dennyverbeeck** [comment]
This will also be fixed by having the history API #261.

**Comment by blootsvoets** [comment]
If #261 would be used for that, it would be good then if the REST API could filter on only source modifications. Preferably, there would be a list of sources. The process should be quite efficient, because it will be applied _every_ time the REST API requests data from a user. This query is used to determine which data belong to the user.

**Comment by dennyverbeeck** [comment]
Closed by #261
